tests/test_oauth2.py

```diff
@@ -420,6 +424,11 @@ def test_request_info_with_req_and_extra_args(self):
         assert h_args == {}
         assert isinstance(cis, AuthorizationRequest)
 
+    def test_request_info_when_request_is_cc_access_token_request(self):
```

Comment:
I would like to see some basic verification of the results as well.

Reply:
Done. See if this is what you were requesting.